Apps/phpanorama/panorama.json

```diff
@@ -173,7 +173,7 @@
         {
             "action": "unblock url",
             "description": "Unblock an URL",
-            "verbose": "This action will remove the URL from the URL Filtering profile that was created/updated in the <b>block url</b> action. The action then proceeds to <b>commit</b> the changes to Panorama, followed by a commit to the device group. If the device group happens to be <b>shared</b>, then a commit will be sent to all the device groups belonging to it.",
+            "verbose": "For Version 8 and below, this action will remove the URL from the URL Filtering profile that was created/updated in the <b>block url</b> action. For Version 9 and above, this action will remove the URL from the Custom URL category that was created/updated in the <b>block url</b> action. The action then proceeds to <b>commit</b> the changes to Panorama, followed by a commit to the device group. If the device group happens to be <b>shared</b>, then a commit will be sent to all the device groups belonging to it.",
```

Comment:
For UI, https://10.1.66.58:9999/docs/app_reference/panorama_3d88e427-0e61-4311-a613-bbddeac532e0#unblock-url

![image](https://user-images.githubusercontent.com/82124762/129092118-14ec36ad-f4e2-4aad-a6a6-258e9a3e3972.png)